chromadb/auth/token_authn/__init__.py

```diff
@@ -232,4 +231,6 @@ def authenticate_or_raise(self, headers: Headers) -> UserIdentity:
         time.sleep(
             random.uniform(0.001, 0.005)
         )  # add some jitter to avoid timing attacks
+        from fastapi import HTTPException
```

Comment:
Do we really want to import in the hot path here? Why do we need this change?

Reply:
It's a good point. I had it initially in the `__init__` as a `self.HTTPException`. Shall I move it back?

Reply:
@HammadB, address this separately here - https://github.com/chroma-core/chroma/pull/2477